ansible/roles/test/tasks/continuous_link_flap/continuous_link_flap_helper.yml

```diff
@@ -0,0 +1,32 @@
+- block:
+    - debug: msg="Interface to flap {{interface}}"
+
+    - name: Shutting down interface
+      become: true
+      shell: ifconfig {{interface}} down
```

Comment:
>ifconfig [](start = 13, length = 8)

use `config` CLI

Reply:
I changed it to ifdown\ifup in V2. I am not aware if we have config CLI command for this which also does not delete IPV6 address, kindly let me know exact CLI to use. Thx.

Reply:
using CLI now.